README.md

```diff
@@ -1,4 +1,15 @@
-# go-spiffe library [![GoDoc](https://godoc.org/github.com/spiffe/go-spiffe?status.svg)](https://godoc.org/github.com/spiffe/go-spiffe)
+# go-spiffe (v1) library [![GoDoc](https://godoc.org/github.com/spiffe/go-spiffe?status.svg)](https://godoc.org/github.com/spiffe/go-spiffe)
+
+# Deprecation Warning
```

Comment:
I was thinking of having this as a level 2 title, not at the same level of the main title of the document, what do you think?
```suggestion
## Deprecation Warning
```

Reply:
I wanted it loud and easily seen. If you think it looks bad, I'd be ok with `##`.

Reply:
I'm fine either way, let's keep it loud :)